### PR TITLE
Add CPurple theme

### PR DESCRIPTION
**Pull request by TheOnlyJoeEnderman**
CPurple is based on CBlue and CGreen. If I need to add any more credit or details please help me, I am new. also, I think Ill shrink the thumbnail a bit real quick to be the same size as the others.

### CODE REVIEW

**Comment by rcalixte** [comment]
@TheOnlyJoeEnderman Do you want to mark this as a draft until you're ready?

**Comment by TheOnlyJoeEnderman** [comment]
It is ready now

File: CPurple/metadata.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "last-edited": 1703441316
+}
```
Comment (rcalixte) [inline CPurple/metadata.json:3]: This file should not be here. Also, you might consider adding a README file. 

This is also missing two key files:
1. You should have a screenshot.png displaying the theme with icons, etc. (Check the Spices site for others.)
2. You should have an info.json with the following contents:
```json
{
    "author": "TheOnlyJoeEnderman"
}
```

**Review by rcalixte — CHANGES_REQUESTED** [review]

File: CPurple/metadata.json (hunk 1 above)
Comment (TheOnlyJoeEnderman) [reply CPurple/metadata.json]: all issues listed plus license have been fixed.

**Review by rcalixte — APPROVED** [review]

**Pull request merged**